promscale/installation/upgrade.md

```diff
@@ -0,0 +1,70 @@
+# Upgrade Promscale
+Promscale consists of the Promscale Connector, and the Promscale PostgreSQL
+extension. When you upgrade your Promscale installation, you need to check both
+components. This section contains important information about upgrading.
+
+The process for updating your Promscale installation is:
+
+1.  Check that the most recent required version of the Promscale extension is
+    available.
```

Comment:
```suggestion
1.  Check that the most recent required versions of PostgreSQL, TimescaleDB and the
Promscale extension are available.
```

Reply:
Can we also provide instructions on how to check the versions of PostgreSQL, TimescaleDB and Promscale?

For TimescaleDB and Promscale this would be. Show current version installed:

`SELECT extname, extversion FROM pg_extension WHERE extname IN ('timescaledb', 'promscale');
`
Show available versions:

`SELECT name, version FROM pg_available_extension_versions WHERE name IN ('timescaledb', 'promscale');`

Reply:
To get the PostgreSQL version:
`SHOW server_version;`